src/pages/persona/styled.ts

```diff
@@ -34,6 +23,21 @@ const Description = styled.p`
   line-height: 18.5px;
 `;
 
+const ResultContainer = styled(Container)`
+  background-color: ${color.grayscalef9};
+`;
+
+const TitleDiv = styled.div`
+  padding: 24px 0 16px 0;
```

Comment:
fyi)

css shorthand;

```suggestion
  padding: 24px 0 16px;
```
이렇게 생략해도 같습니다.